README.md

```diff
@@ -1,21 +1,43 @@
 # Maslow
 
-Maslow is a tool to create and manage needs. It's a Rails app which is
-part of the GOV.UK Publishing Platform.
+Maslow is a tool to create and manage user needs.
 
-## Dependencies
 
-- Ruby
-- Bundler
-- NodeJS
-- Yarn
-- A running instance of the [Publishing API](https://github.com/alphagov/publishing-api)
+## Technical documentation
 
-## User accounts
+This is a Ruby on Rails app, and should follow [our Rails app conventions][].
 
-Authentication is provided by the [GDS-SSO](https://github.com/alphagov/gds-sso) gem, and in the production environment an instance of [Signon](https://github.com/alphagov/signon) must be running in order to sign in.
+To use the app locally you need [GOV.UK Docker][] installed.
+
+[our Rails app conventions]: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html
+[GOV.UK Docker]: https://github.com/alphagov/govuk-docker
+
+### Running the app
+
+Go to the `~/govuk/maslow` folder and run `govuk-docker-up`.  This starts the application and its dependencies.
+
+You should then be able to access maslow in your web browser at http://maslow.dev.gov.uk/
+
+### Testing
+
+
+The default `rake` task runs all the tests and records code coverage:
+
+```sh
+govuk-docker-run bundle exec rake
```

Comment:
I think we've been trying to avoid explicit mention of `govuk-docker-run` (in case the command itself changes in the future) and instead leave it as a more vague "Use GOV.UK to run these commands" (for example https://github.com/alphagov/support/pull/861/commits/faef1a16ed8cb7d77bcd2aa9bc37aa5949af5ec2).